src/main/java/io/getunleash/util/UnleashConfig.java

```diff
@@ -211,6 +221,10 @@ public String getInstanceId() {
         return instanceId;
     }
 
+    public String getConnectionId() {
```

Comment:
I'm probably committing some terrible Java sin here but I'd be tempted to remove the public access modifier here. We've been burned so many times by exposing things that we didn't need to and having someone form a weird dependency on them
```suggestion
    String getConnectionId() {
```

Reply:
Ah good catch. Forgot about package-private access in Java :)